lua/which-key/colors.lua

```diff
@@ -2,20 +2,23 @@ local M = {}
 
 local links = {
   [""] = "Function",
-  Separator = "DiffAdded",
+  Separator = "DiffAdd",
   Group = "Keyword",
   Desc = "Identifier",
   Float = "NormalFloat",
   Value = "Comment",
 }
 
-if vim.fn.hlexists("WhichKeySeperator") then
-  links["Separator"] = "WhichKeySeperator"
-end
-
 function M.setup()
-  for k, v in pairs(links) do
-    vim.api.nvim_command("hi def link WhichKey" .. k .. " " .. v)
+  for k, v in pairs({
```

Comment:
A suggestion: it is better to create a new variable above this for table and use that here.

Reply:
Why?

Reply:
Just a stylistic suggestion and feels more idiomatic. Currently, it is okay, but if later, more groups are added, it would be harder to read, and would most possibly have to do what I suggested. So it is better to do it now. :-)

Reply:
I don't understand why is that will be more idiomatic and why it would be harder to head in the future... Is there any style guidelines which say that?